lib/super_digit.rb

```diff
@@ -1,15 +1,36 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(mn) - m = number of times super_digit is called, n = number of digits in n
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Since you're going to recur roughly the number of digits number of times, the time complexity is O(n^2) where n is the number of digits.